makefiles/Makefile.third_party.win.mk

```diff
@@ -420,9 +423,24 @@ dependencies/install/lib/protobuf.jar: dependencies/install/bin/protoc.exe
 	  ../src/google/protobuf/descriptor.proto
 	cd dependencies\\sources\\protobuf-$(PROTOBUF_TAG)\\java\\core\\src\\main\\java && jar cvf ..\\..\\..\\..\\..\\..\\..\\install\\lib\\protobuf.jar com\\google\\protobuf\\*java
 
-# Clean everything.
-clean_third_party:
+# TODO: TBD: Don't know if this is a ubiquitous issue across platforms...
+# Handle a couple of extraneous circumstances involving TortoiseSVN caching and .svn readonly attributes.
+kill_tortoisesvn_cache:
+	$(TASKKILL) /IM "$(TSVNCACHE_EXE)" /F /FI "STATUS eq RUNNING"
+
+remove_readonly_svn_attribs: kill_tortoisesvn_cache
+	if exist dependencies\sources\* $(ATTRIB) -r /s dependencies\sources\*
+
+
+# Clean everything. Remember to also delete archived dependencies, i.e. in the event of download failure, etc.
+clean_third_party: remove_readonly_svn_attribs
```

Comment:
@lperron I found that the clean step FAILS due to permissions issues in the ``.svn`` directory.